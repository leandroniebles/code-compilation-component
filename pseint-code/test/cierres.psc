subproceso foo
proceso faltan_cerrar
  Si verdadero entonces
  Mientras Verdadero Hacer
  Repetir
  Para i<-1 hasta 10 hacer
  Dimension a(10);
  Para cada x de a hacer
  Segun i hacer
  