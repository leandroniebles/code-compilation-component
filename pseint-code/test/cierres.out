=== Line 1: SynError 119
=== Line 9: SynError 118
=== Line 8: SynError 114
=== Line 6: SynError 114
=== Line 5: SynError 115
=== Line 4: SynError 116
=== Line 3: SynError 117
=== Line 2: SynError 119
